core/src/main/java/com/linecorp/armeria/server/ServerBuilder.java

```diff
@@ -1662,4 +1662,34 @@ public String toString() {
                 clientAddressSources, clientAddressTrustedProxyFilter, clientAddressFilter, clientAddressMapper,
                 enableServerHeader, enableDateHeader);
     }
+
+    ServerConfig buildServerConfig(ServerConfig existingConfig) {
```

Comment:
Could you put this method on top of `toString()`?
We usually put `toString()` at the end. 😄

Reply:
Done.